README.md

```diff
@@ -25,12 +25,23 @@ It provides the following solvers:
 
 Some demonstrations (both in Python and Jupyter Notebook format) are available in the examples folder.
 
+#### Using and citing the toolbox
+
+If you use this toolbox in your research and find it useful, please cite POT using the following bibtex reference:
+```
+@article{flamary2017pot,
```

Comment:
maybe 
@misc{flamary2017pot,
title={POT Python Optimal Transport library},
author={Flamary, R{\'e}mi and Courty, Nicolas},
url={https://github.com/rflamary/POT},
year={2017}
}

is better ?